daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -89,18 +81,18 @@ message NodeExercise {
     repeated string children = 7; // node ids
     repeated string stakeholders = 8;
     repeated string signatories = 9;
-    repeated string controllers = 10;
+    reserved 10; // was controllers
```

Comment:
Is always empty since SDK 1.0.0.

Reply:
Why were those always empty? Because we can re-compute them from them? But don't we need this to compute the informees?

Reply:
Does the following extract from the previous version of  transaction specification help ?

> The ``actors`` field contains the parties that exercised the choice.
>  The ``controllers`` field contains the parties that _can_ exercise
>  the choice. Note that according to the ledger model these two fields
>  _must_ be the same. For this reason the ``controllers`` field was
>  removed in version 6 -- see *since version 6* below.

Reply:
It does indeed. Thanks a lot.